src/styles/_NavBar.scss

```diff
@@ -57,4 +57,32 @@
     border: none;
     cursor: pointer;
   }
+}
+
+@media screen and (max-width: 1005px) { 
+  .NavBar {
+    .ventures-slogan {
+      width: 30%;
+      margin-right: 30%;
+    }
+  } 
+}
+
+@media screen and (max-width: 715px) { 
+  .NavBar {
+    .ventures-slogan {
+      width: 40%;
+      margin-right: 10%;
+    }
+  } 
+}
+
+@media screen and (max-width: 550px) { 
+  .NavBar {
+    order: 2;
+
+    .ventures-slogan {
+      display: none;
+    }
+  } 
```

Comment:
The media queries look amazing so far! I like that you had the NavBar pop to the bottom as a regular phone app would look. The transitions are smooth. It looks like the only thing we will need to add are some queries on the user name/edit button on Profile and stacking the SearchBar/Filters which will be easy!